scripts/listResume.script.js

```diff
@@ -11,12 +11,13 @@ function JobResume(jobs) {
 };
 
 JobResume.prototype.toHtml = function() {
-  var $newJobResume = $('jobHistories.listJobs').clone();
-  $newJobResume.find('jobTitle').html(this.jobTitle);
-  $newJobResume.find('empName').html(this.empName);
-  $newJobResume.find('empLoc').html(this.empLoc);
-  $newJobResume.find('dateEmp').html(this.dateEmp);
-  $newJobResume.find('respon').html(this.respon);
+  var $newJobResume = $('article.listJobs').clone();
```

Comment:
also had to use a removeClass because it kept rendering endlessly line 20 as well as changing the locations where David had put his "template" which is listed under listJobs